internal/pkg/service/buffer/sink/tablesink/storage/volume/registration/config.go

```diff
@@ -0,0 +1,15 @@
+package registration
+
+// Config configures registry of active volumes.
+type Config struct {
+	// TTLSeconds defines number of seconds after the volume registration expires if the node is not available.
+	// The mechanism is implemented via etcd lease read more: https://etcd.io/docs/v3.5/learning/api/#lease-api
+	TTLSeconds int `json:"registrationTimeToLive" configKey:"registrationTTL" configUsage:"Number of seconds after the volume registration expires if the node is not available." validate:"required,min=1,max=60"`
+}
```

Comment:
Created `registration.Config`, this was part of the `VolumesConfig`, but it was wrong.
The `TTLSeconds` is a global configuration and it should NOT be patchable from the Sink configuration.